admin/downloads_manager.php

```diff
@@ -184,6 +191,10 @@
                   '<a href="' . zen_href_link(FILENAME_DOWNLOADS_MANAGER, zen_get_all_get_params(array('padID', 'action')) . 'padID=' . $padInfo->products_attributes_id . '&page=' . $_GET['page'] . '&action=edit') . '" class="btn btn-primary" role="button">' . IMAGE_EDIT . '</a>' .
                   '&nbsp;<a href="' . zen_href_link(FILENAME_ATTRIBUTES_CONTROLLER, 'products_filter=' . $padInfo->products_id . '&current_categories_id=' . $padInfo->master_categories_id) . '" class="btn btn-primary" role="button">' . IMAGE_EDIT_ATTRIBUTES . '</a>'
                 );
+                $contents[] = array('align' => 'center', 'text' => '<a href="'  . zen_href_link(FILENAME_PRODUCT, 'cPath=' . $cPath . '&product_type=' . $padInfo->products_type . '&pID=' . $padInfo->products_id . '&action=new_product&page=' . $_GET['page']) . '" class="btn btn-primary" role="button">' . IMAGE_EDIT_PRODUCT . '</a>'); 
```

Comment:
Should we leave out the `$cPath` parameter here since it's not known in this context?